public/components/Sidebar/Sidebar.js

```diff
@@ -19,7 +19,7 @@ class Sidebar extends React.Component {
     }
 
     return(
-      <div className={className} onClick={() => this.setCampaignStateFilter(stateValue)}>{displayName}</div>
+      <Link to="/campaigns" className={className} onClick={() => {this.setCampaignStateFilter(stateValue)}}>{displayName}</Link>
```

Comment:
It would have to be `onClick={this.setCampaignStateFilter.bind(this, stateValue)}` and I prefer it how it is now! I didn't mean to add those extra brackets though